test/rulesets/Base/base.jl

```diff
@@ -152,11 +152,8 @@
         rrule_test(identity, randn(rng, 4), (randn(rng, 4), randn(rng, 4)))
     end
 
-    @testset "Constants" begin
-        test_scalar(one, 5)
-        test_scalar(one, -4.1)
-
-        test_scalar(zero, 5)
-        test_scalar(zero, -4.1)
+    @testset "Constants" for x in (-0.1, 6.4, 1.0+0.5im, -10.0+0im)
```

Comment:
For symmetry's sake, perhaps add a purely imaginary test?

Reply:
```suggestion
    @testset "Constants" for x in (-0.1, 6.4, 1.0+0.5im, -10.0+0im, 0+200im)
```